modules/cfe_assert/src/cfe_assert_io.c

```diff
@@ -87,6 +87,9 @@ void UT_BSP_DoText(uint8 MessageType, const char *OutputMessage)
                 Prefix = "N/A";
                 break;
             case UTASSERT_CASETYPE_BEGIN:
+                CFE_ES_WriteToSysLog("\n");
+                CFE_ES_WriteToSysLog("\n");
+                CFE_ES_WriteToSysLog("\n"); /* add a bit of extra whitespace between tests */
```

Comment:
This should be left out of this PR ideally - we can do something better in #1266